composer.json

```diff
@@ -16,8 +16,7 @@
     ],
     "license": "MIT",
     "require": {
-        "php": "~5.4",
-        "dominionenterprises/util": "~1.0",
+        "php": "~5.4 || ~7.0",
```

Comment:
I had left this one 5.4 intentionally but at this time I think we can update to 5.6 to be consistent.